### PR TITLE
Nep 97 bug app freeze internet

### PR DESCRIPTION
**Pull request by shrijan-siris**
- [ 97] Include Jira Ticket No. in Title
- [ ] Write Unit Tests
- [ ] Add screenshots whenever possible
- [ ] Make sure Azure Pipelines build succeed
- [ ] Adress Automated Code Reviews from Hound, Codacy and Deepscan
- [x] Add Reviewers

### CODE REVIEW

File: src/components/article-card.component.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { Time } from './time.component'
 
 class CardComponent extends React.PureComponent {
 	handleArticleCardPressed() {
-		this.props.actions.startToOpenArticle(this.props.article)
+		// this.props.actions.startToOpenArticle(this.props.article)
```
Comment (syuraj) [inline src/components/article-card.component.js]: @shrijan-siris is this commented on purpose?

File: src/screens/twitter-screen/twitter.screen.js [hunk 2]
```diff
@@ -1,70 +1,72 @@
...
-			<QueryRenderer
-				environment={environment}
-				query={graphql`
-					query twitterQuery {
-						getTweets {
+const TwitterComponent = ({ navigation }) => {
+	const netInfo = useNetInfo()
+	const [isConnected, setConnected] = useState(true)
+	useEffect(() => {
+		setConnected(netInfo.isConnected)
```
Comment (syuraj) [inline src/screens/twitter-screen/twitter.screen.js]: @shrijan-siris this code is repeated in every Screen, can this be moved to app layout at upper level?

File: src/components/article-card.component.js (hunk 1 above)
Comment (shrijan-siris) [reply src/components/article-card.component.js]: yes as we are not using redux and we are not passing actions i have commented it

File: src/screens/twitter-screen/twitter.screen.js (hunk 2 above)
Comment (shrijan-siris) [reply src/screens/twitter-screen/twitter.screen.js]: that will effect the whole app then , probably

**Pull request merged**